cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -643,7 +643,7 @@ public LocalDocument call(SQLDatabase db) throws Exception {
         return null;
     }
 
-    private BasicDocumentRevision createDocument(SQLDatabase db,String docId, final DocumentBody body)
+    private BasicDocumentRevision createJsonDocument(SQLDatabase db, String docId, final DocumentBody body)
```

Comment:
Good point -- the idea was basically to say that this wasn't doing anything with the attachments only the JSON data itself. I'm not sure Internal or InDatabase get this across, though I'm not sure JSON does either -- any suggestions @rhyshort @tomblench ?